bbj-vscode/src/language/bbj.langium

```diff
@@ -13,7 +13,7 @@ entry Model:
 
 // Main program
 Program:
-    Statements*
+    Statements* EOF
```

Comment:
Why is it necessary?

Reply:
It hides one annoying warning, that this rule could be empty.

Reply:
But it is not the Root rule, so wrong place :D